reporter/src/funTest/kotlin/reporters/NoticeTemplateReporterFunTest.kt

```diff
@@ -91,10 +95,26 @@ private fun generateReport(
     val input = ReporterInput(
         ortResult,
         config,
-        copyrightGarbage = copyrightGarbage
+        copyrightGarbage = copyrightGarbage,
+        licenseConfiguration = createLicenseConfiguration()
     )
 
     val outputDir = createTempDir(ORT_NAME, NoticeTemplateReporterFunTest::class.simpleName).apply { deleteOnExit() }
 
     return NoticeTemplateReporter().generateReport(input, outputDir, options).single().readText()
 }
+
+private fun createLicenseConfiguration(): LicenseConfiguration {
+    val catIncludeNotice = LicenseCategory("include-in-notice-file")
+    val catIncludeSource = LicenseCategory("include-source-code-offer-in-notice-file")
```

Comment:
Nit: We currently do not really use these kind of prefixes (also "lic" below) for variable naming the in the code base, but prefer something like "includeNoticeCategory" or "mitLicense" etc.